tests/expressions_case/tests.py

```diff
@@ -722,6 +722,22 @@ def test_filter_with_aggregation_in_predicate(self):
             transform=itemgetter("integer", "integer2", "max"),
         )
 
+    def test_case_with_extra_kwargs(self):
+        case_expression = Case(
+            When(integer=1, then=Value(10)),
+            When(integer=2, then=Value(20)),
+            default=Value(0),
```

Comment:
`default` is already defined as a keyword argument, so to really test `**extra`, let's use an example that instead of `default` uses a custom keyword.